readme/template-guide-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template Guide
+
+Read this document before you start working on the README template.
+
+
+*Note: As your project evolves and grows, so should your README file. READMEs should be revisited regularly to ensure relevant information is captured for new users.*
```

Comment:
**Suggestion (non-blocking, organization):** I love this note and think it's great! I wonder if it might be better if it came after the last paragraph of the "Why do I need a README?" section from a documentation flow and organization perspective. Your thoughts?

Reply:
I agree! I'll move the note.